app/views/recipes_foods/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class='container' >
+  <div class='loginForm'>
+    <h1>Add Food Ingredients</h1>
+    <table class="add-food-th">
+      <thead>
+        <tr>
+          <th>Food</th>
+          <th>Measurement Unit</th>
+          <th>Unit Price</th>
+          <th>Quantity</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+              <% @foods.each do |food| %>
+                <tr>
+                  <%= form_with model: @recipe_food, url: user_recipe_recipes_foods_path do |f| %> 
+                  <td>  <%= food.name %></td>
+                  <td> <%= food.measurement_unit %></td>
+                  <td> <%= food.price %></td>
+                  <td>  <%= f.number_field :quantity %></td>
+                    <%= f.hidden_field :food_id, value: food.id %>
+                    <td>  <%= f.submit 'Add Items' %></td>
```

Comment:
- [ ] I can't able to add ingredients. when I click on `add items`, nothing happened, Please fix this.

Reply:
@Deepakdanger Thank you for your feedback.